mailbox/store/src/main/java/org/apache/james/mailbox/store/StoreMailboxManager.java

```diff
@@ -217,8 +218,8 @@ public PreDeletionHooks getPreDeletionHooks() {
     /**
      * Generate and return the next uid validity
      */
-    protected int randomUidValidity() {
-        return Math.abs(RANDOM.nextInt());
+    protected UidValidity randomUidValidity() {
```

Comment:
we should have a single way of generating UidValidity objects and it should be  `UidValidity.create()`
We should document that `UidValidity.of` should be used only for deserialization